lib/internal/async_hooks.js

```diff
@@ -159,6 +160,7 @@ function emitHook(symbol, asyncId) {
   // Use a single try/catch for all hook to avoid setting up one per
   // iteration.
   try {
+    // Using var here instead of let because "for (var ...)" is faster than let
```

Comment:
```suggestion
    // Using var here instead of let because "for (var ...)" is faster than let
    // Refs: https://github.com/nodejs/node/pull/30380#issuecomment-552948364
```